arel-helpers.gemspec

```diff
@@ -1,23 +1,26 @@
-$:.unshift File.join(File.dirname(__FILE__), 'lib')
+$LOAD_PATH.unshift File.join(File.dirname(__FILE__), 'lib')
 require 'arel-helpers/version'
 
 Gem::Specification.new do |s|
-  s.name     = "arel-helpers"
+  s.name     = 'arel-helpers'
   s.version  = ::ArelHelpers::VERSION
-  s.authors  = ["Cameron Dutro"]
-  s.email    = ["camertron@gmail.com"]
-  s.homepage = "https://github.com/camertron/arel-helpers"
+  s.authors  = ['Cameron Dutro']
+  s.email    = ['camertron@gmail.com']
+  s.homepage = 'https://github.com/camertron/arel-helpers'
   s.license  = 'MIT'
-  s.description = s.summary = "Useful tools to help construct database queries with ActiveRecord and Arel."
+  s.description = s.summary = 'Useful tools to help construct database queries with ActiveRecord and Arel.'
 
   s.platform = Gem::Platform::RUBY
 
-  s.add_dependency 'activerecord', '>= 3.1.0', '< 7'
+  s.add_dependency 'activerecord', '>= 3.1.0', '< 6.2'
```

Comment:
I'd like to keep 7 as the upper bound here. If rails 6.2 doesn't require us to make any changes, then people can continue using the current version of arel-helpers and it will Just Work™.

Reply:
No problem at all, I will revert.